template.yml

```diff
@@ -34,70 +40,79 @@ Resources:
         Statement:
           - Effect: Allow
             Action: codepipeline:ListPipelines
-            Resource: "*"
+            Resource: '*'
 
   MTTREventHandler:
-    Type: "AWS::Serverless::Function"
+    Type: 'AWS::Serverless::Function'
     Properties:
       Description: Create CloudWatch mttr metric from cloudwatch alarm events
       Handler: mttr.handler
       Runtime: nodejs12.x
       CodeUri: .build/src
+      Environment:
+        Variables:
+          IGNORED_ALARM_NAME_PATTERN: !Ref AlarmNameBlackListPattern
```

Comment:
can we change to `ALARM_NAME_BLACKLIST_PATTERN` - makes it consistent with language elsewhere

Reply:
Done